schemas/styles/csl-choose.rnc

```diff
@@ -11,8 +11,8 @@ div {
   
   ## When specifying conditions, we allow either the attribute syntax or the element-based one.
   choose.condition =
-    (condition.atts, relation?, match?)
-    | element cs:conditions { match?, relation?, condition.elem+ }
+    (condition.atts, relation, match?)
+    | element cs:conditions { match?, relation, condition.elem+ }
```

Comment:
Wait, why is this non-necessary? If I want to test conditions for the base item, I would omit `@relation`.

Reply:
The pattern for relation is this:
```
relation = attribute relation { relations }?
```
So, it's always optional.

Reply:
Okay

Reply:
Here's the definition for `@form` (in `names.form`)

```
 attribute form { "long" | "short" | "count" }?
```

On `cs:names` it appears only as `names.form`. No need to make it optional twice.